miio/devicestatus.py

```diff
@@ -119,6 +119,7 @@ def embed(self, other: "DeviceStatus"):
         other_name = str(other.__class__.__name__)
 
         self._embedded[other_name] = other
+        setattr(self, other_name, other)
```

Comment:
We can add this (and update the docstring to inform anyone reading the docs about the fact), and also change the separator to `__` instead of `:`, in this PR?

Reply:
No `__` can not be used as seperator, I tried it but very bad things start happening.
Python uses __ for python variables like __main__, __name__ and even worse __getattribute__ itself is also an attribute to the class, so you get into an infinite loop of __getattribute__ getting itself using another __getattribute__ and you crash the program.

Reply:
I don't think having a double undescore should be a problem as a separator in the middle of the name, as far as I know the dunder methods need to both start and end with `__`.